src/cmds/pbsdsh.c

```diff
@@ -95,6 +107,96 @@ bailout(int sig)
  * @return - Void
  *
  */
+
+
+int
```

Comment:
You separated the block comment for wait_for_task() from the function itself.  Please put it back on top of the function.  Also, please add a doxygen header comment for host_match()

Reply:
Yeah, my bad, will fix it.